backend/djangoindia/api/views/user.py

```diff
@@ -35,25 +59,45 @@ def partial_update(self, request, *args, **kwargs):
             return Response(UserMeSerializer(user).data, status=status.HTTP_200_OK)
         except serializers.ValidationError as e:
             return Response(
-                {
-                    "message": e.detail,
-                },
+                {"message": e.detail},
                 status=status.HTTP_400_BAD_REQUEST,
             )
-        except Exception as e:
+        except Exception:
             return Response(
-                {
-                    "message": "Something went wrong",
-                },
+                {"message": "Something went wrong"},
```

Comment:
Something like this would be better
```suggestion
                {"message": "Something went wrong:" + str(e)},
```